src/pages/Thanks/ThanksPage.vue

```diff
@@ -1,10 +1,10 @@
 <template>
 	<www-page data-testid="thanks-page">
-		<template v-if="iwdHeaderExpEnabled">
-			<iwd-thanks-page-variations
```

Comment:
Go ahead and remove these old IWD components, if they aren't used anywhere. We can look up GH history, if we need to go back.